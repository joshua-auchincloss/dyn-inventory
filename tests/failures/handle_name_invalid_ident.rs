// invalid identifier for handle_name
//
//
//
//
//
//
//
//
//

dyn_inventory!(
    TraitB: StructB<T> {
        a: bool,
        t: T,
    };
    handle_name = My-Handle,
);

fn main() {}
