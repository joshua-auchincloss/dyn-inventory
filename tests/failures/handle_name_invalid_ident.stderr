error: cannot find macro `dyn_inventory` in this scope
  --> tests/failures/handle_name_invalid_ident.rs:12:1
   |
12 | dyn_inventory!(
   | ^^^^^^^^^^^^^
   |
   = note: `dyn_inventory` is in scope, but it is a crate, not a macro
help: consider importing this macro
   |
12 + use dyn_inventory::dyn_inventory;
   |
